.github/workflows/ci.yml

```diff
@@ -282,10 +282,12 @@ jobs:
       matrix:
         target:
           - runner-type: gpu
-            iree-ctest-label-regex: ^requires-gpu|^driver=vulkan$|^driver=cuda$
+            iree-ctest-extra-args:
+              --label-regex ^requires-gpu|^driver=vulkan$|^driver=cuda$
+              --label-exclude ^device-count=2$
             iree-nvidia-sm80-tests-disable: 1
```

Comment:
The latest CI failures suggest that nvidia-sm80 tests are running when they shouldn't be... https://github.com/openxla/iree/actions/runs/5670372477/job/15365761577?pr=14418#step:7:1160

`/work/runtime/src/iree/hal/drivers/cuda/native_executable.c:102: INTERNAL; CUDA driver error 'CUDA_ERROR_INVALID_PTX' (218): a PTX JIT compilation failed;`

```
The following tests FAILED:
	 11 - iree/tests/e2e/collectives/collectives_test_single_gpu (Failed)
	 12 - iree/tests/e2e/collectives/collectives_test_multi_gpu (Failed)
	 72 - iree/tests/e2e/matmul/e2e_matmul_direct_f32_gpu_large_LLVMGPUMatmulTensorCore_cuda_cuda (Failed)
	 73 - iree/tests/e2e/matmul/e2e_matmul_direct_f32_gpu_large_unaligned_cuda_cuda (Failed)
	 74 - iree/tests/e2e/matmul/e2e_matmul_direct_f16_gpu_large_unaligned_cuda_cuda (Failed)
	 75 - iree/tests/e2e/matmul/e2e_matmul_direct_f32_gpu_large_mma_sync_LLVMGPUMatmulTensorCoreMmaSync_cuda_cuda (Failed)
	 76 - iree/tests/e2e/matmul/e2e_matmul_direct_f16_gpu_large_LLVMGPUMatmulTensorCore_cuda_cuda (Failed)
	 77 - iree/tests/e2e/matmul/e2e_matmul_direct_f16_gpu_large_mma_sync_LLVMGPUMatmulTensorCoreMmaSync_cuda_cuda (Failed)
	734 - iree/tests/transform_dialect/cuda/double_mma_layout_analysis.mlir.test (Failed)
	735 - iree/tests/transform_dialect/cuda/mma_elemwise_layout_analysis.mlir.test (Failed)
	736 - iree/tests/transform_dialect/cuda/mma_reduction_layout_analysis.mlir.test (Failed)
	737 - iree/tests/transform_dialect/cuda/mma_using_layout_analysis.mlir.test (Failed)
```

Maybe some issue with newlines in the ctest command coming from this.

Reply:
I added a new flag to exclude multi-device tests due to the behavior of multiple ctest `--label-exclude` parameters.